src/index.js

```diff
@@ -9,13 +10,25 @@ var shisha = {
      * @param data
      * @param locals
      * @param callback
+     * @param capath
      */
-    smoke: function (data, locals, callback) {
+    smoke: function (data, locals, callback, capath) {
```

Comment:
`ca` is optional and it's an argument, like `locals`. The suggestion could be to have a new `options` object and `locals` as its attribute, eg:
```js
var options = {
   locals: {...},
   ca: ...
} 
```
and the smoke signature will change into:

```js
smoke: function (data, options, callback)
```

wdyt? @unlucio ?